logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -139,6 +139,10 @@ public long getCurrentByteSize() {
         return this.currentByteSize;
     }
 
+    public long getCurrentPhysicallyPersistedByteSize() {
```

Comment:
not sure why we use physically in `getCurrentPhysicallyPersistedByteSize`? why not just `getCurrentPersistedByteSize`?